src/plugins/chat.ts

```
@@ -424,10 +424,39 @@ async function prepareMessages(
     }
+
+    let historyNewMessages = recentMessage
+    if (
+        config.modelCompletionCount > 0 &&
+        temp.lastHistoryNew &&
+        temp.lastHistoryNew.length > 0
+    ) {
+        let overlap = Math.min(
+            temp.lastHistoryNew.length,
+            recentMessage.length
+        )
```

ESLint

[error] Replace `⏎············temp.lastHistoryNew.length,⏎············recentMessage.length⏎········` with `temp.lastHistoryNew.length,·recentMessage.length`

(prettier/prettier)